bot/seasons/evergreen/movie.py

```diff
@@ -0,0 +1,198 @@
+import logging
+import random
+from enum import Enum
+from typing import Any, Dict, List, Tuple
+from urllib.parse import urlencode
+
+from aiohttp import ClientSession
+from discord import Embed
+from discord.ext.commands import Bot, Cog, Context, group
+
+from bot.constants import Tokens
+from bot.pagination import ImagePaginator
+
+# Define base URL of TMDB
+BASE_URL = "https://api.themoviedb.org/3/"
+
+logger = logging.getLogger(__name__)
+
+# Define movie params, that will be used for every movie request
+MOVIE_PARAMS = {
+    "api_key": Tokens.tmdb,
+    "language": "en-US"
+}
+
+
+class MovieGenres(Enum):
+    """Movies Genre names and IDs."""
+
+    Action = "28"
+    Adventure = "12"
+    Animation = "16"
+    Comedy = "35"
+    Crime = "80"
+    Documentary = "99"
+    Drama = "18"
+    Family = "10751"
+    Fantasy = "14"
+    History = "36"
+    Horror = "27"
+    Music = "10402"
+    Mystery = "9648"
+    Romance = "10749"
+    Science = "878"
+    Thriller = "53"
+    Western = "37"
+
+
+class Movie(Cog):
+    """Movie Cog contains movies command that grab random movies from TMDB."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.http_session: ClientSession = bot.http_session
+
+    @group(name='movies', aliases=['movie'], invoke_without_command=True)
+    async def movies(self, ctx: Context, genre: str = "", amount: int = 5) -> None:
+        """
+        Get random movies by specifying genre. Also support amount parameter, that define how much movies will be shown.
+
+        Default 5. Use .movies genres to get all available genres.
+        """
+        # Check is there more than 20 movies specified, due TMDB return 20 movies
+        # per page, so this is max. Also you can't get less movies than 1, just logic
+        if amount > 20:
+            await ctx.send("You can't get more than 20 movies at once. (TMDB limits)")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 movies. Just logic.")
```

Comment:
"movies" should not be plural.
```suggestion
            await ctx.send("You can't get less than 1 movie. Just logic.")
```

Reply:
The second sentence also feels unneeded and maybe a bit rude.